src/server.js

```diff
@@ -43,15 +43,17 @@ app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
 // serve swagger
-app.get('/doc', (req, res) => {
-  res.send(swaggerSpec);
-});
+// app.get('/doc', (req, res) => {
```

Comment:
Please remove commented code or uncomment it.

Reply:
This has been resolved